### PR TITLE
[launch_mysql_service] Per second graph functionality & driver script.

### PR DESCRIPTION
**Pull request by sspano**
@stfeng2 @gareth-ferneyhough

### CODE REVIEW

File: perfkitbenchmarker/scripts/database_scripts/plot_scatter_points.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+# This variable controls the verticle lines inside the chart.
+#   0 means no vertical lines.
+Y_HEIGHT = '3000'
+Y_TICS = '100'
+X_INTERVAL = '1200'
+ITERATIONS = '9'
+
+
+class gnuplot_info():
```
Comment (gareth-ferneyhough) [inline perfkitbenchmarker/scripts/database_scripts/plot_scatter_points.py]: class names should be CamelCase

File: perfkitbenchmarker/scripts/database_scripts/file_to_plot.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  """Plotter generates a per second output graph of TPS vs Thread Values.
+  Given run configurations, and run stderr filenames for any number of PKB runs,
+  Plotter extracts TPS values and generates a gnuplot graph which can be
+  uploaded to cloud storage.
+  """
+
+  def __init__(self, run_seconds, report_interval, filename=None,
+               cloud_storage=None):
+    """Args:
+
```
Comment (gareth-ferneyhough) [inline perfkitbenchmarker/scripts/database_scripts/file_to_plot.py]: remove blank line and indent each arg line

File: perfkitbenchmarker/scripts/database_scripts/file_to_plot.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  def add_file(self, filename):
+    """Given STDERR filename for ONE run with a given thread count, add data.
+
+    Args:
+      filename: (string) Name of file to be parsed.
+
+    Raises:
+      STDERRFileDoesNotExistError:
+    """
+    # TODO: Open file (try/catch)
```
Comment (gareth-ferneyhough) [inline perfkitbenchmarker/scripts/database_scripts/file_to_plot.py]: remove todo if not needed

File: perfkitbenchmarker/scripts/database_scripts/file_to_plot.py [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+    Raises:
+      PatternNotFoundError.
+    """
+    tps_values = []
+    line = f.readline()
+    while line:
+      if line.strip() == DATA_INDICATOR_LINE:
+        line = f.readline()  # blank line
+        # Data collection phase: self._entries lines of collection.
```
Comment (gareth-ferneyhough) [inline perfkitbenchmarker/scripts/database_scripts/file_to_plot.py]: i dont really understand this comment

File: perfkitbenchmarker/scripts/database_scripts/plot_scatter_points.py [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+  def __init__(self, gnuplot_data_filename):
+    self.output_dir = "./charts"
+    self.output_chart = self.output_dir + "/innodb_pages.png"
```
Comment (gareth-ferneyhough) [inline perfkitbenchmarker/scripts/database_scripts/plot_scatter_points.py]: i would use os.path.join to join paths as it is more clear what your intent is. also don't have to worry about ensuring your forward slashes are correct

**Comment by stfeng2** [comment]
@sspano the "rebase" introduced a significant amount of changes that should not become part of this change. We need to revert the "rebase" and only check-in to master the changes related to this particular issue.

**Comment by sspano** [comment]
@gareth-ferneyhough @stfeng2 I have addressed @gareth-ferneyhough 's comments locally. I have copied the 2 most recent commits from this PR to a new branch and plan on submitting a new PR after a manual test. I will then cancel this PR.
I will copy @gareth-ferneyhough 's comments to the new PR so they are not lost but they have been taken into consideration with the new PR. 

Thanks for your patience with this one

**Pull request closed without merging**